contrib/pkg/broker/server/server.go

```diff
@@ -146,19 +164,30 @@ func (s *server) bind(w http.ResponseWriter, r *http.Request) {
 	if result, err := s.controller.Bind(instanceID, bindingID, &req); err == nil {
 		util.WriteResponse(w, http.StatusOK, result)
 	} else {
-		util.WriteResponse(w, http.StatusBadRequest, err)
+		util.WriteErrorResponse(w, http.StatusBadRequest, err)
 	}
 }
 
 func (s *server) unBind(w http.ResponseWriter, r *http.Request) {
 	instanceID := mux.Vars(r)["instance_id"]
 	bindingID := mux.Vars(r)["binding_id"]
+	q := r.URL.Query()
+	serviceID := first(q["service_id"])
```

Comment:
serviceID := q.Get("service_id")
there's no need for first, as Get already returns the first value.